numpydoc/numpydoc.py

```diff
@@ -113,8 +113,11 @@ def mangle_signature(app, what, name, obj, options, sig, retann):
 
     if not hasattr(obj, '__doc__'):
         return
-
-    doc = SphinxDocString(pydoc.getdoc(obj))
+    try:
+        doc = SphinxDocString(pydoc.getdoc(obj))
+    except ValueError as e:
+        filename = obj.__code__.co_filename
```

Comment:
Also, the full object name should in principle contained in `name` --- does it get shown by Sphinx?